index.js

```diff
@@ -641,36 +643,46 @@ function normalizeArguments(url, opts) {
 	return opts;
 }
 
-function got(url, opts) {
-	try {
-		const normalizedArgs = normalizeArguments(url, opts);
+function create(defaults = {}) {
+	function got(url, opts) {
+		try {
+			opts = extend(true, {}, defaults, opts);
```

Comment:
It looks like the global defaults are not inherited? Was that intentional? My expectation would be that when I create a client, the global defaults are inherited.

I think you can easily address by renaming the function argument to `clientDefaults` and making this line:

```js
opts = extend(true, {}, defaults, clientDefaults, opts);
```

Reply:
the global defaults are inherited as the top-level `got` export dogfoods this interface and all public `create()` calls operate off of that instance:
```js
const got = create(defaults);  // L43
```

Reply:
heads up that, unless I'm missing something, this should be covered by the _preserve global defaults_ test case in `test/create.js` line 18